src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -10,30 +10,25 @@
 
 namespace System.IO.Packaging.Tests
 {
-    public class Tests
+    public class Tests : FileCleanupTestBase
     {
         private const string Mime_MediaTypeNames_Text_Xml = "text/xml";
         private const string Mime_MediaTypeNames_Image_Jpeg = "image/jpeg"; // System.Net.Mime.MediaTypeNames.Image.Jpeg
         private const string s_DocumentXml = @"<Hello>Test</Hello>";
         private const string s_ResourceXml = @"<Resource>Test</Resource>";
 
-        private static FileInfo GetFileSavedWithGuidName(string test, string name)
+        private FileInfo GetFileSavedWithGuidName(string existingFileName)
         {
-            var fiDoc = new FileInfo(name);
-            var ba = File.ReadAllBytes(fiDoc.FullName);
-            var fiGuidName = new FileInfo("Temp-" + Guid.NewGuid().ToString().Replace("-", "") + fiDoc.Extension);
-            File.WriteAllBytes(fiGuidName.FullName, ba);
-			// If a test leaves a Test-* file in existence, then uncomment following line to determine which test it was.
-			// Console.WriteLine("{0}:{1}", test, fiGuidName.Name);
-            return fiGuidName;
+            var existingDoc = new FileInfo(existingFileName);
+            var content = File.ReadAllBytes(existingDoc.FullName);
+            var newFile =  new FileInfo($"{GetTestFilePath()}.{existingDoc.Extension}");
+            File.WriteAllBytes(newFile.FullName, content);
+            return newFile;
         }
 
-        public static FileInfo GetGuidNameForNewFile(string test, string extension)
+        public FileInfo GetGuidNameForNewFile(string extension)
         {
-            var fiDoc = new FileInfo("Temp-" + Guid.NewGuid().ToString().Replace("-", "") + extension);
-			// If a test leaves a Test-* file in existence, then uncomment following line to determine which test it was.
-			// Console.WriteLine("{0}:{1}", test, fiDoc.Name);
-			return fiDoc;
+            return new FileInfo($"{GetTestFilePath()}.{extension}");
```

Comment:
One of the benefits of GetTestFilePath is it includes in the file name information about the test being used, in case the file gets left behind or there's an issue with it, or whatnot.  That's done using Caller attributes, which are filled in by the compiler based on the call site.  But that scheme is then defeated somewhat when the GetTestFilePath is in a helper method rather than the actual test.  You might consider adding such Caller attribute arguments to these helpers and then passing the state through explicitly to GetTestFilePath.